NL/2DCode/raw/123.json

```diff
@@ -0,0 +1,52 @@
+{
+    "JSON": {
+        "ver": "1.0.0",
+        "nam": {
+            "fn": "'t Achternaam",
+            "fnt": "T<ACHTERNAAM",
+            "gn": "V\u00f6ornaam",
+            "gnt": "VOORNAAM"
+        },
+        "dob": "",
+        "v": [
+            {
+                "tg": "840539006",
+                "vp": "1119305005",
+                "mp": "CoronaVac",
+                "ma": "ORG-100032020",
+                "dn": 8,
+                "sd": 9,
+                "dt": "2021-02-18",
+                "co": "GR",
+                "is": "Ministry of Health Welfare and Sport",
```

Comment:
Again, I suspect MinVWS is not the issuer for GR?

Reply:
MinVWS could be the issuer of a dgc, based on a doctor's vaccination proof, based on a shot actually taken in GR. See above comment.